catalog/imaging/imaging-transformer-nitf/src/test/java/org/codice/alliance/transformer/nitf/image/ImageInputTransformerTest.java

```diff
@@ -145,27 +145,6 @@ public void testNitfParsing() throws Exception {
   }
 
   @Test
-  public void testHandleMissionIdSuccessful() throws Exception {
-    Metacard metacard = metacardFactory.createMetacard("missionIdTest");
-    transformer.handleMissionIdentifier(metacard, "0123456789ABC");
-    assertThat(metacard.getAttribute(Isr.MISSION_ID).getValue(), is("789A"));
-  }
-
-  @Test
-  public void testHandleMissionIdEmpty() throws Exception {
-    Metacard metacard = metacardFactory.createMetacard("noMissionIdTest");
-    transformer.handleMissionIdentifier(metacard, "0123456");
-    assertThat(metacard.getAttribute(Isr.MISSION_ID), nullValue());
-  }
-
-  @Test
-  public void testHandleMissionIdNoImageIdentifier() throws Exception {
-    Metacard metacard = metacardFactory.createMetacard("noIdentifierTest");
-    transformer.handleMissionIdentifier(metacard, null);
-    assertThat(metacard.getAttribute(Isr.MISSION_ID), nullValue());
-  }
```

Comment:
These tests are no longer needed now that the mission.id `Attribute` was moved to https://github.com/codice/alliance/blob/e7c036fa8a556d1175a55e69bcba6b64ca4ecbae/catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/image/ImageAttribute.java.

Reply:
❓ There is an `ImageAttributeTest.java` class, should we add a test for the MissionID in there?